### PR TITLE
fix(DDI-1337): Allow for initial table sorting

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

**Comment by Radha-Alberta** [comment]
- [ ] Issue in React: Table data is not loaded on Page load for Safari browsers and Iphone

Iphone_Safari
Iphone_Chrome
Mac_Web_Safari_Desc
Mac_Web_Safari_Asc

File: libs/web-components/src/components/table/TableSortHeader.svelte [hunk 1]
```diff
@@ -21,43 +21,62 @@
...
+  }
+
+  goa-icon {
+    scale: 0.8;
+  }
 
-  img.asc {
-    transform: rotate(180deg);
+  .direction--none goa-icon {
+    height: 10px;
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/table/TableSortHeader.svelte:59]: Maybe we can change it to 'rem'

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**